internal/pkg/plugin/trellogithub/workflow.go

```diff
@@ -18,10 +18,10 @@ var trelloWorkflow = &github.Workflow{
 
 // Options is the struct for configurations of the trellogithub plugin.
 type Options struct {
-	Owner       string
-	Org         string
-	Repo        string
-	Branch      string
+	Owner       string `validate:"required_without=Org"`
+	Org         string `validate:"required_without=Owner"`
+	Repo        string `validate:"required"`
+	Branch      string `validate:"required"`
 	BoardId     string
```

Comment:
I think these three IDs are mandatory, too. This might have been a mistake earlier. Please also fix it.

Reply:
ok,I will work for it.

Reply:
Maybe take a look at what errors were there before and I will fix them. @IronCore864

Reply:
and there are four IDs

Reply:
Let's tackle this in a separate issue.